src/lib.rs

```diff
@@ -96,12 +100,19 @@ impl Browser {
 
     /// Returns true if this specific browser is detected in the system
     pub fn exists(&self) -> bool {
-        open_browser_with_options(
-            *self,
-            "https://rootnet.in",
-            BrowserOptions::new().with_dry_run(true),
-        )
-        .is_ok()
+        #[cfg(not(target_os = "ios"))]
```

Comment:
Handle this via the `BrowserOptions.dry_run` to use the same code pattern as in other platforms. See [android as an example](https://github.com/amodm/webbrowser-rs/blob/main/src/android.rs#L23)